test/Azure.Functions.Cli.Tests/E2E/CreateFunctionTests.cs

```diff
@@ -402,5 +402,34 @@ await CliTester.Run(new RunConfiguration
                 }
             }, _output);
         }
+
+        [Fact]
+        public async Task create_httpTrigger_configAuthLevel_powershell()
+        {
+            await CliTester.Run(new RunConfiguration
+            {
+                Commands = new[]
+                {
+                    "init . --worker-runtime powershell",
+                    "new --template \"HttpTrigger\" --name MyHttpTriggerFunction --authlevel Anonymous -a"
+                },
+                CheckFiles = new FileResult[]
+                {
+                    new FileResult
+                    {
+                        Name = Path.Combine("MyHttpTriggerFunction", "function.json"),
+                        ContentContains = new []
```

Comment:
It looks like the root logic for `ContentContains` is a `StringComparison.Ordinal` which means this should confirm the right behavior.